database/Reserve.js

```diff
@@ -3,14 +3,16 @@ const db = require('./index.js');
 mongoose.Promise = global.Promise;
 
 const reserveSchema = new mongoose.Schema({
-  restaurantId: Number,
-  tableId: Number,
+  restaurant: {
+    $ref: 'restaurants',
+    $id: 
+  },
   date: Date,
   time: Date,
   chairs: Number,
   reserved: {type: Boolean, default: false}
 });
 
-const Reserve = mongoose.model('Reserve', reserveSchema);
+const Reservation = mongoose.model('Reserve', reserveSchema);
 
-module.exports = Reserve;
\ No newline at end of file
+module.exports = Reservation;
```

Comment:
@pryz25 @jfeng3124 File should end with a newline character (see red arrow above). AirBnB style guide linting should point this out. The newline character helps certain command line tools display files in a readable fashion.